### PR TITLE
fix: mf-3930 use scoped chain id

### PR DESCRIPTION
**Pull request by UncleBill**
https://mask.atlassian.net/browse/MF-3930

### CODE REVIEW

**Comment by hanszhoou** [comment]
Testing solana

File: packages/web3-hooks/base/src/useContext.tsx [hunk 1]
```diff
@@ -58,43 +60,35 @@ export function NetworkContextProvider({ value, children }: React.ProviderProps<
...
-    const [account, setAccount] = useState<string>()
-    const [chainId, setChainId] = useState<Web3Helper.ChainIdAll>()
-    const [networkType, setNetworkType] = useState<Web3Helper.NetworkTypeAll>()
-    const [providerType, setProviderType] = useState<Web3Helper.ProviderTypeAll>()
+    const [_account, setAccount] = useState<string>()
+    const [_chainId, setChainId] = useState<Web3Helper.ChainIdAll>()
+    const [_networkType, setNetworkType] = useState<Web3Helper.NetworkTypeAll>()
+    const [_providerType, setProviderType] = useState<Web3Helper.ProviderTypeAll>()
+
+    const account = (controlled ? value.account : _account) ?? globalAccount
```
Comment (guanbinrui) [inline packages/web3-hooks/base/src/useContext.tsx]: Please keep the same logic as the previous version when `controlled` is false.

File: packages/mask/src/plugins/Tips/contexts/Tip/TipTaskProvider.tsx [hunk 2]
```diff
@@ -78,6 +78,7 @@ export const TipTaskProvider: FC<React.PropsWithChildren<Props>> = memo(({ child
         },
         [key],
     )
+    console.log('tips', { targetChainId, tokenMap, nativeTokenDetailed })
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Tips/contexts/Tip/TipTaskProvider.tsx]:
```suggestion
```

**Comment by hanszhoou** [comment]
LGTM for Switching between Solana and EVM.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**